### PR TITLE
Convert  "Tonc's Text Engine" to markdown

### LINKED ISSUE #61: Convert to markdown : tte (Tonc's Text Engine)

### PR DESCRIPTION
**Pull request by mtthgn**
The main thing to note in this PR is that there were some parts of the text where a specific section or subsection number is referenced. With the removal of section numbers as part of the converstion to MDBook, I wasn't sure how we've resolved these in the past, particularly from the perspective of minimizing content changes. I'll call these spots out in the PR.

Closes #61

### CODE REVIEW

File: content/tte.md [hunk 1]
```diff
@@ -0,0 +1,2253 @@
...
+  </table>
+</div>
+
+First, note the great differences in values: from hundreds for the tilemaps and objects to _thousands_ in the case of bitmaps and tile renderers. And this is per character, so writing large swats of text can lead to significant slowdown.
+
+The `null()` renderer is a dummy renderer, used to find the overhead of the TTE system. 200 isn't actually that bad, all things considered (remember: ROM code). That said, now compare this number to the regular tilemap time: the overhead is takes up a significant fraction of the time here. Also note the difference between the standard and 8×16 versions of `se_drawg`: this is purely due the loops
+
+Half of the TTE overhead actually comes from the wrapping code; cursor setting and checking can be relatively slow. And I'm not even considering clipping here.
+
+For the bitmap and tile renderers, I've timed three versions. A ‘base’ version, using the template from `chr4_drawg_b1cts_base()` in section 22.6.2.; C-optimized versions, which are the default renderers; and a fast asm version.
```
Comment (mtthgn) [inline content/tte.md:2241]: We need to resolve the reference to "section 22.6.2", which is talking about the subsection with the id of `#ssec-chr-drawg`.

**Review by mtthgn — COMMENTED** [review]
Let me know how y'all think we should resolve these references, and I'll get the PR updated!

**Comment by mtthgn** [comment]
@avivace whoops! Looks like I missed two of those. I'll fix those and resolve conflicts now.

**Comment by mtthgn** [comment]
Alright @avivace comments are now addressed.

**Comment by mtthgn** [comment]
Alright @avivace, the direct section and subsection references have been replaced with section names + anchor tags  like we discussed in Discord. Sorry this small change took so long to get done! Busy time of year 😅

**Review by avivace — APPROVED** [review]
Alright, let's go!

**Pull request merged**